docs/docs/faq/question_03.md

```diff
@@ -0,0 +1,48 @@
+# My simulator is not picklable, can I still use multiprocessing?
+
+Yes, but you will have to make a few adjustments to your code. 
+
+Some background: when using `num_workers > 1`, you might experience an error like:
+```
+TypeError: xyz: Only Vector instance can be pickled
```

Comment:
This is specific to Neuron. Perhaps we should just write that one might experience errors that certain objects cannot be pickled?